openedx/core/djangoapps/content/search/tests/test_documents.py

```diff
@@ -420,6 +423,7 @@ def test_html_published_library_block(self):
                 "display_name": "Text 2",
                 "description": "This is a Test",
             },
+            "publish_status": "published",
```

Comment:
(If we end up keeping this field) could you please add a test for `"publish_status": "modified"`?